plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/command/edit/EditCommandsPresenter.java

```diff
@@ -142,6 +142,11 @@ public void apply(WorkspaceDto arg) throws OperationException {
                 fetchCommands();
                 fireConfigurationUpdated(selectedConfiguration);
             }
+        }).catchError(new Operation<PromiseError>() {
+            @Override
+            public void apply(PromiseError arg) throws OperationException {
+                dialogFactory.createMessageDialog("Error", arg.toString(), null).show();
```

Comment:
plz, use arg.getMessage() instead of arg.toString()
because toString() returns nothing interesting for the user